analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -244,6 +246,9 @@ def __parse_compiler_includes(lines):
 
         include_paths = []
 
+        if not lines:
```

Comment:
Do we need this? `for line in lines.splitlines():` will just iterate 0 times for an empty string so we will end up returnin `include_paths` anyways.

Reply:
This is an extra check in case of `None` is the input not a string for some reason. splitlines will fail on `None`.